docker-compose/docker-compose.yaml

```diff
@@ -46,14 +46,26 @@ services:
     hostname: nginx
     volumes:
       - ./nginx/nginx.conf:/etc/nginx/nginx.conf:ro
-      - ./nginx/conf.d/:/etc/nginx/conf.d/:ro
+      - ./nginx/proxy_params:/etc/nginx/proxy_params:ro
+      - ./nginx/conf.d/:/etc/nginx/conf.d/
+      - ./nginx/templates/:/etc/nginx/templates/:ro
       - /srv/buildbot/packages:/srv/buildbot/packages:ro
       - /srv/buildbot/galera_packages:/srv/buildbot/galera_packages:ro
       - /srv/buildbot/helper_files:/srv/buildbot/helper_files:ro
+      - /etc/letsencrypt/live:/etc/nginx/ssl:ro
```

Comment:
Let's add a comment here so that we make sure that we move this in `/srv/buildbot` (or somewhere else) when we move the renew routine from the host to containers.

Reply:
Sure!